tests/kernel/mem_protect/syscalls/src/main.c

```diff
@@ -318,6 +319,39 @@ void test_more_args(void)
 		      "syscall didn't match impl");
 }
 
+/**
```

Comment:
@bbolen this looks more like a unit test for the cortex-r architecture; i wonder if it could reside as a standalone unit test under tests/arch/arm

Reply:
agreed.  i've moved it and updated the PR